### ISSUE TITLE
multi-sheet not compatible with access control

### ISSUE THREAD

**Issue by cbrunet** [audreyt/ethercalc#167]
Hi,

The view mode of multi sheets is broken. If I try to display /=mysheet/view , or event /=mysheet.1/view, toolbars are displayed. Furthermore, /=mysheet/view display the index of subsheets, instead of displaying the first sheet.

This makes access control (i.e. --key parameter) unusable with multisheet.

Charles.

**Comment by audreyt** [comment]
Yes, that is indeed the case; we'd need to write a separate view/edit control for the `multi/` code. This is also tied to #146 as only the multi-sheet UI is exposed to Sandstorm users.
